rest/rest-api/src/main/java/org/apache/brooklyn/rest/api/PolicyApi.java

```diff
@@ -36,6 +36,8 @@
 @Api("Entity Policies")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
+/** @deprecated since 0.12.0 use AdjunctApi */
```

Comment:
Shouldn't this be "since 0.13.0"?

(Please search https://github.com/apache/brooklyn-server/pull/821/files for that text)